docs/docs/reference/release-notes/v3.0/index.md

```diff
@@ -0,0 +1,271 @@
+Please refer to React's own [release notes](https://reactjs.org/blog/2020/10/20/react-v17.html) for a full list of changes.
+
+The minimum version of Gatsby is now 16.9.0 to support Fast Refresh, React Hooks, and Suspense by default. We've also made sure we're 100% compatible with React 17. To use the new [React JSX transform](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html) have a look at the [`babel-preset-gatsby`](#babel-preset-gatsby) section below.
+
+With this change, we'll be adding more experiments to support [Concurrent mode](https://reactjs.org/blog/2019/11/06/building-great-user-experiences-with-concurrent-mode-and-suspense.html) and [React Server Components](https://reactjs.org/blog/2020/12/21/data-fetching-with-react-server-components.html) in future releases.
+
+## GraphQL 15
+
+Please refer to `graphql-js`'s own [release notes](https://github.com/graphql/graphql-js/releases/tag/v15.0.0) for a full list of changes.
+
+With this upgrade we can finally leverage interface inheritance for [queryable interfaces](https://www.gatsbyjs.com/docs/reference/graphql-data-layer/schema-customization/#queryable-interfaces-with-the-nodeinterface-extension).
+You no longer need to use `@nodeInterface` directive in schema customization:
+
+```diff
+exports.createSchemaCustomization = function createSchemaCustomization({ actions }) {
+  const { createTypes } = actions
+  createTypes(`
+-   interface Foo @nodeInterface
++   interface Foo implements Node
+    {
+      id: ID!
+    }
+  `)
+}
+```
+
+Also, Gatsby now displays GraphQL deprecations as CLI warnings when queries. Example output:
+
+![GraphQL deprecation warning in CLI](./graphql-deprecation-warnings.png)
+
+## ESLint 7
+
+Please refer to `eslint`'s own [migration guide](https://eslint.org/docs/user-guide/migrating-to-7.0.0) to update your custom ESLint files.
+
+If you rely on Gatsby's default ESLint configuration, you should have a smooth transition. We upgraded the underlying rules, so you might get some new warnings/errors.
+
+Gatsby no longer uses the deprecated `eslint-loader`. We’ve moved to `eslint-webpack-plugin`. By using the plugin, behaviour has changed a little bit, as warnings and errors are displayed later than in version 2.
+
+## gatsby-plugin-image@1.0.0
+
+This release includes a new plugin, `gatsby-plugin-image`, replacing the old `gatsby-image` plugin. Built from the ground up for speed and best practices, it's the most performant way to use images in React. If you saw your scores drop with recent Lighthouse updates, migrating to this plugin should help you get top scores again. It comes with a new, simplified GraphQL API (no more fragments), as well as a new `StaticImage` component that's as easy to use as an `<img>` tag.
+
+The `GatsbyImage` component supports three responsive layout types, as shown in the video below.
+
+<video controls autoplay loop>
+  <source
+    type="video/mp4"
+    src="/docs/reference/built-in-components/layouts.mp4"
```

Comment:
@ascorbic Is this the right path? The video for it is in this folder

Reply:
Oh, I see, the video got added again in another path? 🤔

Reply:
Oh oops. Yes, I meant to change that! I was going to update the other video instead of uplaoding a new one here